packages/react/src/components/MultiSelect/MultiSelect.featureflag.stories.js

```diff
@@ -21,6 +21,29 @@ export default {
       </WithFeatureFlags>
     ),
   ],
+  argTypes: {
+    size: {
+      options: ['sm', 'md', 'lg'],
+      control: { type: 'select' },
+    },
+    direction: {
+      options: ['top', 'bottom'],
+      control: { type: 'radio' },
+    },
+    type: {
+      options: ['inline', 'default'],
+      control: { type: 'radio' },
+    },
+    disabled: {
+      control: { type: 'boolean' },
+    },
+    invalid: {
+      control: { type: 'boolean' },
+    },
+    warn: {
+      control: { type: 'boolean' },
+    },
+  },
```

Comment:
```suggestion
  },
    parameters: {
    controls: {
      exclude: [
        'filterItems',
        'translateWithId',
        'titleText',
        'open',
        'selectedItems',
        'itemToString',
        'itemToElement',
        'locale',
        'items',
        'id',
        'initialSelectedItems',
        'sortItems',
        'compareItems',
        'downshiftProps',
      ],
    },
  },
```

We can also add some control filtering to clean up the control panel a bit like the other stories